CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Fabric Test Maintainers
-*	@bmos299 @christo4ferris @denyeart @dongmingh @lhaskins @mastersingh24 @scottz64 @adnan-c
+*	@hyperledger/fabric-test-maintainers
```

Comment:
What is this?  Where is fabric-test-maintainers defined? 
Does it have anything to do with removing gerrit references from the MAINTAINERS.rst file?

Reply:
I think Ry accidentally used a wrong title for the commit. He has opened these for all of the hyperledger repos, and a couple had this wrong title, so that will need to be updated. Regardless, all of the repos have a team defined that actually give privileges on the repo to it's maintainers. So Everytime we update the maintainers we also need to update the team and the codeowners file. With this change we will now only have to update the team, you can find the fabric-test teams defined here: https://github.com/hyperledger/fabric-test/settings/collaboration 

Thanks Scott

Reply:
That "collaboration" link does not work for me, @btl5037 . Maybe we are missing a file? And I still don't see where the list of fabric-test maintainers will be kept, once you delete it here. When we remove or add someone to our CODEOWNERS list, are you saying we will also need to ask @ryjones  to edit some list somewhere outside of our fabric-test repo?

Reply:
I removed the CODEOWNERS change